disk/disk.go

```
@@ -153,6 +159,9 @@ func (dc *DiskCollector) GetMetricTypes(cfg plugin.Config) ([]plugin.Metric, err
 func (dc *DiskCollector) CollectMetrics(mts []plugin.Metric) ([]plugin.Metric, error) {
 	metrics := []plugin.Metric{}
 
+	ignoreLoopback, _ = mts[0].Config.GetBool("ignore_loopback")
+	ignoreRAM, _ = mts[0].Config.GetBool("ignore_ram")
```

Is it ok to ignore error here?

---

Yes. Even these config params are not set by user, Snap framework will do that with default values. That mans that all metrics incoming to CollectMetrics will have these params set.